pgcli/pgexecute.py

```diff
@@ -260,7 +260,11 @@ def run(self, statement, pgspecial=None, exception_formatter=None,
                     cur = self.conn.cursor()
                     try:
                         for result in pgspecial.execute(cur, sql):
-                            yield result + (sql, True)
+                            # e.g. execute_from_file already appends these
```

Comment:
I agree this is a bit ugly, but I'm not sure what else can be done here. I'm trying to think of a way to communicate from execute_from_file() that we successfully ran the query and got all the result tuple. But I can't think of a great way to do it.